### ISSUE TITLE
bodhi-client in epel8

### ISSUE THREAD

**Issue by onosek** [fedora-infra/bodhi#4019]
Hi,
is there any plan to release bodhi-client v5 in epel8? It would be great to have it there. It will help me release fedpkg for epel8.
I suppose, that the same proposal for epel7 is more complicated, right? But epel8 itself is helpful though.
Thanks an have a nice day.

**Comment by cverna** [comment]
> Hi,
> is there any plan to release bodhi-client v5 in epel8? It would be great to have it there. It will help me release fedpkg for epel8.
> I suppose, that the same proposal for epel7 is more complicated, right? But epel8 itself is helpful though.
> Thanks an have a nice day.

Hi,
There is currently no plan for that  but I think that is enforcing the idea to create a dedicated repo and package for bodhi-client which will make it easier to package in epel8.

**Comment by onosek** [comment]
>> There is currently no plan for that but I think that is enforcing the idea to create a dedicated repo and package for bodhi-client which will make it easier to package in epel8.

I agree that separate bodhi-client would make releasing compatible versions easier.
I expected that the release will not happen soon and I (hopefully) resolved conflicting versions ... for now.
